src/main/kotlin/cn/yiiguxing/plugin/translate/trans/deepl/DeeplTranslator.kt

```diff
@@ -171,10 +181,10 @@ object DeeplTranslator : AbstractTranslator(), DocumentationTranslator {
     }
 
     private fun translateDocumentation(documentation: String, srcLang: Lang, targetLang: Lang): String {
-        return SimpleTranslateClient(this, DeeplTranslator::call, DeeplTranslator::parseTranslation).execute(
-            documentation,
-            srcLang,
-            targetLang
-        ).translation ?: ""
+        return SimpleTranslateClient(
+            this,
+            { _, _, _ ->call(documentation, srcLang, targetLang, true) },
+            DeeplTranslator::parseTranslation
+        ).execute(documentation, srcLang, targetLang).translation ?: ""
```

Comment:
The cache key should be updated before calling the `execute` method to avoid cache key conflicts:
```
SimpleTranslateClient.updateCacheKey { it.update("DOCUMENTATION".toByteArray()) }
```

Reply:
Thanks. Corrected.